src/context/FoxEthProvider/FoxEthProvider.tsx

```diff
@@ -275,19 +321,26 @@ export const FoxEthProvider = ({ children }: FoxEthProviderProps) => {
       foxFarmingOpportunities,
       lpLoading,
       farmingLoading,
+      foxFarmingTotalBalanceInBaseUnit: bnOrZero(foxFarmingTotalCryptoAmount)
+        .times(bn(10).pow(lpAssetPrecision))
+        .toString(),
       onOngoingTxIdChange: handleOngoingTxIdChange,
+      lpTokenPrice: bnOrZero(lpTokenPrice).toNumber(),
```

Comment:
Keep it a string and convert to `bnOrZero(lpTokenPrice).toNumber()` in `useBalanceChartData`.
Nothing wrong from a functional perspective, this is safe (price will not be a number higher than `Number.MAX_SAFE_INTEGER`), numbers are primitives and they serialize well etc, but for consistency we tend to pass strings over.
By passing numbers over, we're spreading a pattern where someone could copy x implementation (which works because it's a safe int) to implement y (which is a big number > `Number.MAX_SAFE_INTEGER`)